kong/api/crud_helpers.lua

```diff
@@ -1,7 +1,30 @@
-local utils = require "kong.tools.utils"
-local cjson = require "cjson"
-local responses = require "kong.tools.responses"
-local app_helpers = require "lapis.application"
+local utils         = require "kong.tools.utils"
+local cjson         = require "cjson"
+local responses     = require "kong.tools.responses"
+local app_helpers   = require "lapis.application"
+
+
+local decode_base64 = ngx.decode_base64
+local encode_base64 = ngx.encode_base64
+local encode_args   = ngx.encode_args
+local tonumber      = tonumber
+local ipairs        = ipairs
+local next          = next
+local type          = type
+
+
+local function post_process_row(row, cb)
+  if type(cb) == "function" then
```

Comment:
do you have a suggestion? If you don't do that, then you need to move `if type(cb) == "function" then` to get, post, patch and put. So that's why I did put this repetition there. First one protects against a loop which is good.